java/test/jmri/jmrix/loconet/loconetovertcp/LnTcpServerFrameTest.java

```diff
@@ -40,9 +40,8 @@ public void testGetInstance() {
     public void setUp() {
         JUnitUtil.setUp();
         JUnitUtil.resetProfileManager();
-        lnis = new LocoNetInterfaceScaffold();
         memo = new LocoNetSystemConnectionMemo();
-        lnis.setSystemConnectionMemo(memo);
+        lnis = new LocoNetInterfaceScaffold(memo);
```

Comment:
Be sure to tear down lnis here.  We’ve discovered that JUnit 4 wopill hold on to a reference to it otherwise.

Reply:
That should have been tied to tearDown in both test classes.